### ISSUE TITLE
Radim: Seamless scheduling of flight/team

### ISSUE THREAD

**Issue by kolaf** [airsports-no/airsportslivetracking#33]
Related to #32,  having to register both team and scheduling a flight  Is counterintuitive.

Why do I have to register first to the contest and only after that I can schedule a flight? From the user point of view it should be sufficient to do this in one step. If I want to schedule a flight but I'm not registered as participant in the contest already, it should ask me for a team/aircraft/club details.

**Comment by kolaf** [comment]
The flight scheduling process has been improved and streamlined as part of the new frontend. Please open a new ticket if you have issues or suggestions.
